### PR TITLE
MSP-3: Make home page more polite.

### PR DESCRIPTION
**Pull request by Sophist-UK**
I thought about changing it from "Piss off!" to "Fuck off!" though.

### CODE REVIEW

File: mbspotify/views.py [hunk 1]
```diff
@@ -12,7 +12,21 @@
...
+                '<script type="text/javascript">'
+                    'window.location.href = "https://github.com/metabrainz/mbspotify"'
+                '</script>'
+            '</head>'
+            '<body>'
+                'If you are not automatically redirected, please click ' 
+                '<a href="https://github.com/metabrainz/mbspotify">here</a>.'
+            '</body>'
+        '</html>'
+    )
```
Comment (gentlecat) [inline mbspotify/views.py]: HTML should go into a template file. http://flask.pocoo.org/docs/0.12/tutorial/templates/

File: mbspotify/views_test.py [hunk 2]
```diff
@@ -54,11 +54,6 @@ def create_app(self):
         ))
         return app
 
-    def test_index(self):
-        response = self.client.get("/")
-        # Index page should ask users to piss off.
-        self.assertIn("Piss off!", str(response.data))
```
Comment (gentlecat) [inline mbspotify/views_test.py]: Instead this should test that template is being rendered without errors by comparing the HTTP status to 200.

**Comment by gentlecat** [comment]
@brainzbot, ok to test.

**Comment by mwiencek** [comment]
Why not return an actual 302? Or does it need a fancy "Redirecting you to..." page?

File: mbspotify/views.py (hunk 1 above)
Comment (Sophist-UK) [inline mbspotify/views.py]: This is a temporary "go away" - presumably if this is ever turned into a formal MeB project, there will be a proper home page.

But I will see what i can do.

File: mbspotify/views_test.py (hunk 2 above)
Comment (Sophist-UK) [inline mbspotify/views_test.py]: Agreed. I will try to work out how to do it.

**Comment by Sophist-UK** [comment]
@mwiencek Because I only agreed to fix this because I reported the rude response and Robert asked me to - and I have no idea how to achieve it without research which I don't have time to do at present.

**Comment by mwiencek** [comment]
Returning a 302 with flask will honestly take less time/research than creating a template, but up to you.

**Comment by gentlecat** [comment]
Merged #36 instead.

**Comment by Sophist-UK** [comment]
No problem.

**Comment by gentlecat** [comment]
Thank you!

**Pull request closed without merging**